rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -885,6 +920,11 @@ def test_latest_upgrade_adds_remove_tables(user_data_dir):
     views_before = {x[0] for x in result}
 
     last_db.logout()
+
+    # The last copy of the database has been populated with the assets copied from the globaldb
+    # during the migration 34_45. Take a clean copy of it to execute the upgrade
+    _use_prepared_db(user_data_dir, base_database)
```

Comment:
wait why? This sounds like a solution to a problem that should have been fixed elsewhere

Reply:
For this one since is a generic test I would keep the `_use_prepared_db` between the load of the old database and the new one. Patched the function in the test for v34